SSA/Projects/LLVMRiscV/tests/LLVMIR/preprocess.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
On most systems what you wrote is perfectly fine; it just so happens that my system is weird and has bash in a non-standard location, so it's better to lookup bash with `/usr/bin/env`